go.mod

```diff
@@ -2,6 +2,7 @@ module github.com/cloudquery/plugin-sdk/v4
 
 go 1.21.1
 
+replace github.com/cloudquery/plugin-pb-go => /Users/benbernays/Documents/GitHub/plugin-pb-go
```

Comment:
You can replace this with `plugin-pb-go` `c8efe8ac52377434221d17800afba8eaba57414c` and get the linting/tests working for other things.